client/src/components/GroceryList/GroceryList.js

```diff
@@ -33,48 +23,47 @@ const GroceryList = () => {
     getRecipesInWeek();
   }, []);
 
-  // gibt mir beide Recipe-Objekte:
-  const filterRecipe = recipeObject.filter((item) => item.Recipe);
-  console.log(filterRecipe);
+  const oneShoppingListArray = recipeObject.flatMap((ingredient) => {
+    return ingredient.Recipe.Ingredients;
+  });
 
-  // das ertse Objekt allein, das matched:
-  const findIngredients = filterRecipe.find((item) => item.Recipe.Ingredients);
-  console.log(findIngredients);
-
-  // destructuring funktioniert:
-  // const { RecipeName } = findIngredients;
-  // console.log(RecipeName);
-  // const doubleArray = filterRecipe.Recipe;
-  // console.log(doubleArray);
-
-  const Array = findIngredients.Recipe.Ingredients;
-  console.log(Array);
-
-  // Für jedes Objekt in filterRecipe will ich die Ingredients darausgelesen haben.
-  // also für jedes Object einmal Array (Z.50) machen und dann unten mappen
-  // let arrayLength = recipeObject.length;
+  var holder = {};
```

Comment:
Would stay with `const` or `let` to keep true to your coding style :)

Reply:
oh yes, you're right!:)